.github/workflows/mlc_config.json

```diff
@@ -23,6 +23,9 @@
     },
     {
       "pattern": "^https://www.microsoft.com/edge"
+    },
+    {
+      "pattern": "^https://dev.mysql.com/downloads/shell/"
```

Comment:
Why add this one?

Reply:
The link checking is not stable due to this URL, then ignore it first.